cms/djangoapps/contentstore/views/assets.py

```diff
@@ -591,7 +592,7 @@ def _get_asset_json(display_name, content_type, date, location, thumbnail_locati
     Helper method for formatting the asset information to send to client.
     '''
     asset_url = StaticContent.serialize_asset_key_with_slash(location)
-    external_url = settings.LMS_BASE + asset_url
+    external_url = configuration_helpers.get_value('LMS_ROOT_URL', settings.LMS_ROOT_URL) + asset_url
```

Comment:
is this used in this file? no, wait, you added the import. i'm curious how you found this?

Reply:
@jansenk, I stole this from some LMS XBlock config code here: https://github.com/edx/edx-platform/blob/072b6b8875e2a6323b6666998a3bf5c5e6444039/openedx/core/djangoapps/xblock/apps.py#L59-L64

Reply:
@jansenk - objections to closing this thread?